include/core/utilities.h

```diff
@@ -585,4 +585,28 @@ value_string_to_tensor(const std::string &value_string_0,
   return output_tensor;
 }
 
+/**
+ * @brief Computes equivalent cell diameter by comparing the area to a disk (2D)
+ * or the volume to a sphere (3D).
+ *
+ * @tparam dim Number of spatial dimensions (2D or 3D).
+ *
+ * @param[in] cell_measure Area (2D) or volume (3D) of the cell.
+ *
+ * @param[in] fe_degree Polynomial degree of the shape function.
+ *
+ * @return Cell diameter value.
+ */
+template <int dim>
+inline double
+compute_cell_diameter(const double cell_measure, const unsigned int fe_degree)
+{
+  double h;
+  if constexpr (dim == 2)
+    h = std::sqrt(4. * cell_measure / numbers::PI) / fe_degree;
+  else if constexpr (dim == 3)
+    h = std::pow(6. * cell_measure / numbers::PI, 1. / 3.) / fe_degree;
+  return h;
```

Comment:
```suggestion
  if constexpr (dim == 2)
    return std::sqrt(4. * cell_measure / numbers::PI) / fe_degree;
  else if constexpr (dim == 3)
    return std::cbrt(6. * cell_measure / numbers::PI, 1. / 3.) / fe_degree;
    else
    Assert("....")
```
The code does not support 1D anyway right now so might as well assert if the wrong number is called instead of returning something without an initial value.